lib/zstd.h

```diff
@@ -182,6 +182,7 @@ ZSTDLIB_API unsigned    ZSTD_isError(size_t code);          /*!< tells if a `siz
 ZSTDLIB_API const char* ZSTD_getErrorName(size_t code);     /*!< provides readable string from an error code */
 ZSTDLIB_API int         ZSTD_minCLevel(void);               /*!< minimum negative compression level allowed */
 ZSTDLIB_API int         ZSTD_maxCLevel(void);               /*!< maximum compression level available */
+ZSTDLIB_API int         ZSTD_defaultCLevel(void);           /*!< default compression level, specified by ZSTD_CLEVEL_DEFAULT */
```

Comment:
Note : this is "stable" API domain.

However simple this symbol might be, its introduction within "stable" implies `v1.5.0`.

Reply:
Ah okay, I'll move it to experimental and we can put this on the list of stuff for 1.5